contracts/V1/StateChannelManagerEvents.sol

```diff
@@ -4,36 +4,40 @@ import "./types/DisputeTypes.sol";
 import "./types/DataTypes.sol";
 
 interface StateChannelManagerEvents {
-    event BlockCalldataPosted(
-        bytes32 indexed channelId,
-        bytes32 indexed commitmentHash,
-        address sender,
-        SignedBlock signedBlock,
-        uint256 timestamp
-    );
-    event SetState(bytes32 indexed channelId, bytes encodedState, bytes32 forkId, uint256 timestamp);
+    event StateSnapshotUpdated(bytes32 indexed channelId, StateSnapshot stateSnapshot, uint256 timestamp);
+
+    event BlockCalldataPosted(bytes32 indexed channelId, address sender, SignedBlock signedBlock, uint256 timestamp);
```

Comment:
why remove `bytes32 indexed commitmentHash` you need it https://github.com/peer3to/state-channels-plus/blob/4d778e5905c7505ff3a57753ae99b72b51c7215b/src/stateManager/ValidationService.ts#L397